src/Facebook/InstantArticles/Elements/InstantArticle.php

```diff
@@ -396,6 +396,7 @@ public function addMetaProperty($property_name, $property_content)
 
     public function render($doctype = '<!doctype html>', $format = false)
     {
+        $doctype = is_null($doctype) ? '<!doctype html>' : $doctype;
```

Comment:
@sakatam This would be a breaking change if somebody is already using the default method call (using the default value) which leads to a `<!doctype html>` addition by default. What do you think about it?